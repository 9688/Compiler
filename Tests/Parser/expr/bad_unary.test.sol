Ошибка в строке 1 позиции 2: требуется выражение


