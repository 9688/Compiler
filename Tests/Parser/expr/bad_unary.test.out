└──''
Ошибка синтаксиса в строке 1 позиции 2: требуется выражение


